### PR TITLE
mgr/dashboard: Add documentation about supported browsers

### PR DESCRIPTION
**Pull request by tspmelo**
Fixes: http://tracker.ceph.com/issues/27207

Signed-off-by: Tiago Melo <tmelo@suse.com>

### CODE REVIEW

**Comment by tchaikov** [comment]
retest this please.

File: src/pybind/mgr/dashboard/README.rst [hunk 1]
```diff
@@ -55,6 +55,12 @@ See the `Ceph Dashboard plugin documentation
 and configure the dashboard manually and how to configure other settings, e.g.
 access to the Ceph object gateway.
 
+Supported browsers
+------------------
+
+We support the current versions of `Chrome <https://www.google.com/chrome/>`_
```
Comment (votdev) [inline src/pybind/mgr/dashboard/README.rst]: Do we have any developers which are using Windows IE and can confirm that the UI properly works? If so, maybe we should add this browser here, too.

Comment (runsisi) [inline src/pybind/mgr/dashboard/README.rst]: you mean the latest version? i do think this is reasonable for enterprise users.
i think we need to list the versions we are to support explicitly!

**Comment by zy751713126** [comment]
retest this please

File: src/pybind/mgr/dashboard/README.rst (hunk 1 above)
Comment (LenzGr) [reply src/pybind/mgr/dashboard/README.rst]: I agree that we probably need to be more explicit. For starters, I'd suggest to list the browser versions that we actually work with as part of the development. We could indicate that later versions of these should also work.

In my case, that'd be "Mozilla Firefox 61.0.2 (Linux, 64-bit)" and "Chromium 68.0.3440.75 (Linux, 64-bit)".

Comment (LenzGr) [reply src/pybind/mgr/dashboard/README.rst]: To my knowledge, @Exotelis uses Windows - he will be back next week to comment.

File: src/pybind/mgr/dashboard/README.rst [hunk 2]
```diff
@@ -55,6 +55,16 @@ See the `Ceph Dashboard plugin documentation
 and configure the dashboard manually and how to configure other settings, e.g.
 access to the Ceph object gateway.
 
+Supported browsers
+------------------
+
+We support `Chrome 68+ <https://www.google.com/chrome/>`_ and
```
Comment (LenzGr) [inline src/pybind/mgr/dashboard/README.rst]: I'm a bit wary about the term "support" here - what does this mean?

I wonder if it'd be more appropriate to simply state the browser versions that the dashboard is known to work with. Would it be prudent to mention the browsers' OS platforms as well?
I'd hope that this should not make a difference.

"The Ceph Manager Dashboard" is tested and developed on the following browsers..."

File: src/pybind/mgr/dashboard/README.rst [hunk 3]
```diff
@@ -55,6 +55,16 @@ See the `Ceph Dashboard plugin documentation
...
+`Firefox 61+ <http://www.mozilla.org/firefox/>`_, as both these browsers are
+frequently used for testing and development.
+
+While not officially supported, other browsers like Edge 42+, Opera 41+ and
```
Comment (LenzGr) [inline src/pybind/mgr/dashboard/README.rst]: Dito here about "official support" - I'd rather state that we don't explicitly test on these browsers, but they should work in theory (and that feedback is welcome, if issues are observed).

**Pull request merged**